tracing.go

```diff
@@ -22,32 +22,37 @@ package tchannel
 
 import (
 	"fmt"
-	"math/rand"
+	"net"
+	"strconv"
+	"time"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+	"golang.org/x/net/context"
```

Comment:
ok. I don't get the logic of this particular order, but since it's already everywhere I'l update my changes.